test/unit/trend_test.rb

```diff
@@ -0,0 +1,30 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+require File.join(File.dirname(__FILE__), 'apipie_resource_mock')
+
+require 'hammer_cli_foreman/trend'
+
+describe HammerCLIForeman::Trend do
```

Comment:
Although :heart: for adding the tests, but If we add new commands, we create the functional tests instead of unit. This is because we rely on apidocs we're generating commands from. If the docs were changed somehow in the future we can easily see that something is broken. It might require updating `test/data` as well.

Reply:
Got it, I will move it to functional tests.